tests/client/test_client.py

```diff
@@ -8,6 +8,18 @@ class ApiClientTest(unittest.TestCase):
     def setUp(self):
         self.client = MarklogicApiClient("", "", "", False)
 
+    @patch("caselawclient.Client.MarklogicApiClient._send_to_eval")
+    @patch("caselawclient.Client.decode_multipart")
+    def test_judgment_exists(self, mock_decode, mock_eval):
+        mock_decode.return_value = "true"
+        assert self.client.judgment_exists("/2029/eat/1") is True
```

Comment:
Since we mock `decode_multipart`, on top of not testing the xquery, we could remove the `_send_to_eval` call from this function and the test would still pass.

To mitigate that, we could of course assert that `mock_eval` got called with appropriate params and then that decode also got called with its return value. The downside with that is that's essentially testing implementation rather than function.

To reduce the mocking and asserting to just one function we could separate out the code responsible for sending the request to Marklogic and parsing the response into a separate function. 

```
def judgment_exists(self, judgment_uri: str) -> bool:
    uri = self._format_uri_for_marklogic(judgment_uri)
    content = self._send_judgment_exists_request(uri)

    if content == "true":
        return True
    if content == "false":
        return False
    raise RuntimeError("Marklogic response was neither true nor false")

def _send_judgment_exists_request(self, uri: str) -> str:
    vars: query_dicts.JudgmentExistsDict = {
        "uri": uri,
    }
    response = self._send_to_eval(vars, "judgment_exists.xqy")
    return decode_multipart(response)
```

Even so, we'd still be testing implementation to an extent but seems like a good middle ground for now. In the long run we could maybe consider splitting this class out so that the the calls to marklogic are handled by a separate, `MarklogicQueryExecutor` class that can then be injected into the creation of the `Client` class. That way we don't need to mock `_send_to_eval` but instead create a mock MarklogIcQueryExecutor, and have that class's public methods tested separately.

**Alternatively, we could just make these integration tests, which, if we're not testing the xquery files anywhere else separately, probably makes the most sense.**